modeltestSDK/client.py

```diff
@@ -310,3 +320,12 @@ def delete(self, resource: str = None, endpoint: str = None, parameters: dict =
             Request response
         """
         return self._do_request("DELETE", resource=resource, endpoint=endpoint, parameters=parameters)
+
+    @staticmethod
+    def clear_cache():
+        """
+        Removes cached datapoints (from mtdb.sqlite at local cache folder)
+        """
+        with requests_cache.enabled(cache_name='mtdb', backend='sqlite', use_cache_dir=True,
+                                    expire_after=timedelta(days=30)):
```

Comment:
Repeated config from `do_request()`. Can we configure the cached request once?

Reply:
Move config to config.py. I think the with enabled(): method used each time is the most efficient way to get the correct settings. Otherwise you easily end up applying caching to all requests